ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
     // Add options here
+    'ember-test-selectors': {
+      strip: false,
```

Comment:
Do we want to keep the test selectors in prod build?

Ref: https://github.com/simplabs/ember-test-selectors#configuration

Reply:
I guess No.

Reply:
Then, should this not be `true`?

Reply:
Make sense!